### PR TITLE
Add Computeruniverse, iLoveIMG, iLovePDF, Flathub & Reverso

### PR DESCRIPTION
**Pull request by JustMarkov**
Add Computeruniverse (as hard), iLoveIMG (as hard), iLovePDF (as hard), Flathub (as easy) & Reverso (as hard)

Notes languages: en & ru

P.S.: Separated iLoveIMG and iLovePDF 'cause of different domains and contact forms. Not sure if the accounts are shared, tho.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -9421,6 +9444,28 @@
...
+    },
+
+    {
+        "name": "iLovePDF",
+        "url": "https://www.ilovepdf.com/contact",
+        "difficulty": "hard",
+        "notes": "Fill out the form, selecting the topic \"Privacy\", in which request the deletion of your account.",
+        "notes_ru": "Заполните форму, выбрав тему \"Конфиденциальность\", в который запросите удаление вашего аккаунта.",
+        "domains": [
+            "ilovepdf.com"
```
Comment (Kefaku) [inline _data/sites.json:9465]:
```suggestion
            "ilovepdf.com",
            "developer.ilovepdf.com"
```

File: _data/sites.json [hunk 2]
```diff
@@ -9421,6 +9444,28 @@
...
     },
 
+    {
+        "name": "iLoveIMG",
+        "url": "https://www.iloveimg.com/contact",
+        "difficulty": "hard",
+        "notes": "Fill out the form, selecting the topic \"Privacy\", in which request the deletion of your account.",
+        "notes_ru": "Заполните форму, выбрав тему \"Конфиденциальность\", в который запросите удаление вашего аккаунта.",
+        "domains": [
+            "iloveimg.com"
```
Comment (Kefaku) [inline _data/sites.json:9454]:
```suggestion
            "iloveimg.com",
            "developer.iloveimg.com"
```

File: _data/sites.json [hunk 3]
```diff
@@ -16253,6 +16298,23 @@
...
     },
 
+    {
+        "name": "Reverso",
+        "url": "https://www.reverso.net/contact",
+        "difficulty": "hard",
+        "notes": "On your own, you can only disable your account in your profile settings. If you want to request its removal, you must contact support using the linked form, but they may ask you to confirm your identity.",
+        "notes_ru": "Самостоятельно вы можете только отключить свой аккаунт в настройках профиля. Если вы хотите запросить его удаление, то вы должны обратиться в поддержку, используя форму ниже, но они могут попросить от вас подтверждения личности.",
+        "domains": [
+            "reverso.net",
```
Comment (Kefaku) [inline _data/sites.json:16308]:
```suggestion
            "reverso.net",
            "coniugazione.reverso.net",
            "conjugacao.reverso.net",
            "conjugador.reverso.net",
            "conjugator.reverso.net",
            "conjugueur.reverso.net",
```

File: _data/sites.json [hunk 4]
```diff
@@ -16253,6 +16298,23 @@
...
+            "context.reverso.net",
+            "synonyms.reverso.net",
+            "glagol.reverso.net",
+            "documents.reverso.net",
+            "dictionary.reverso.net",
```
Comment (Kefaku) [inline _data/sites.json:16313]:
```suggestion
            "diccionario.reverso.net",
            "dicionario.reverso.net",
            "dictionary.reverso.net",
            "dictionnaire.reverso.net",
            "dizionario.reverso.net",
```

File: _data/sites.json [hunk 5]
```diff
@@ -16253,6 +16298,23 @@
...
 
+    {
+        "name": "Reverso",
+        "url": "https://www.reverso.net/contact",
+        "difficulty": "hard",
+        "notes": "On your own, you can only disable your account in your profile settings. If you want to request its removal, you must contact support using the linked form, but they may ask you to confirm your identity.",
+        "notes_ru": "Самостоятельно вы можете только отключить свой аккаунт в настройках профиля. Если вы хотите запросить его удаление, то вы должны обратиться в поддержку, используя форму ниже, но они могут попросить от вас подтверждения личности.",
+        "domains": [
+            "reverso.net",
+            "context.reverso.net",
```
Comment (Kefaku) [inline _data/sites.json:16309]:
```suggestion
            "context.reverso.net",
            "corporate-translation.reverso.com",
```

File: _data/sites.json [hunk 6]
```diff
@@ -16253,6 +16298,23 @@
...
+            "synonyms.reverso.net",
+            "glagol.reverso.net",
+            "documents.reverso.net",
+            "dictionary.reverso.net",
+            "grammar.reverso.net"
```
Comment (Kefaku) [inline _data/sites.json:16314]:
```suggestion
            "grammar.reverso.net",
            "konjugator.reverso.net",
            "woerterbuch.reverso.net"
```

**Review by Kefaku — CHANGES_REQUESTED** [review]
There are some missing domains.

**Comment by JustMarkov** [comment]
@Kefaku thanks for your suggestions! I added all of your commits.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for all these new entries!

**Pull request merged**